src/libxtp/gwbse/gwbse.cc

```diff
@@ -555,10 +560,11 @@ bool GWBSE::Evaluate() {
         "You want no GW calculation but the orb file has no QPcoefficients for "
         "BSE");
   }
-  TCMatrix_gwbse Mmn;
+  TCMatrix_gwbse Mmn(*_pLog);
   // rpamin here, because RPA needs till rpamin
   Mmn.Initialize(auxbasis.AOBasisSize(), _gwopt.rpamin, _gwopt.qpmax,
                  _gwopt.rpamin, _gwopt.rpamax);
+  Mmn.SetGPUStreams(_max_gpu_streams);
```

Comment:
yeah I am not sure, I would love to hide the global GPU behaviour in a global variable, because it is sort of like OPENMP threads? What do you think?